### PR TITLE
Sqlite support

### LINKED ISSUE #192: [SQLite] Implement integration

### PR DESCRIPTION
**Pull request by andrwqa**
Added SQLite Adapter

Closes #192

### CODE REVIEW

File: odd_collector/domain/plugin.py [hunk 1]
```diff
@@ -296,6 +296,11 @@ class CouchbasePlugin(BasePlugin):
     num_sample_values: Optional[int] = 10
 
 
+class SQLitePlugin(BasePlugin):
+    data_source: str
```
Comment (Vixtir) [inline odd_collector/domain/plugin.py:300]: Use path type from pydantic for validations https://docs.pydantic.dev/latest/usage/types/#standard-library-types

File: odd_collector/adapters/sqlite/repository/sqlalchemy_repository.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            table = Table(
+                name=table_name,
+                columns=self._get_columns(table_name),
+            )
+            yield table
+
+    def get_views(self) -> Iterable[View]:
+        names = self._inspector.get_view_names()
+
+        views: Dict[str, View] = {
```
Comment (Vixtir) [inline odd_collector/adapters/sqlite/repository/sqlalchemy_repository.py:45]: Do you need dict here? later you just use values from that object

File: odd_collector/adapters/sqlite/adapter.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+from odd_collector.domain.plugin import SQLitePlugin
+from .mappers.database import map_database
+from .mappers.table import map_table
+from .mappers.view import map_view
+from .repository.base_repository import Repository
+from .repository.sqlalchemy_repository import SqlAlchemyRepository
+
+
+class Adapter(AbstractAdapter):
```
Comment (Vixtir) [inline odd_collector/adapters/sqlite/adapter.py:16]: Use BaseAdapter instead for derivation. Take a look at postgres adapter

**Review by Vixtir — CHANGES_REQUESTED** [review]

File: odd_collector/adapters/sqlite/adapter.py (hunk 3 above)
Comment (andrwqa) [reply odd_collector/adapters/sqlite/adapter.py]: Fixed

File: odd_collector/adapters/sqlite/repository/sqlalchemy_repository.py (hunk 2 above)
Comment (andrwqa) [reply odd_collector/adapters/sqlite/repository/sqlalchemy_repository.py]: Fixed

File: odd_collector/domain/plugin.py (hunk 1 above)
Comment (andrwqa) [reply odd_collector/domain/plugin.py]: Added appropriate type.

**Review by Vixtir — APPROVED** [review]

**Pull request merged**